### PR TITLE
[Fix](Expr&code-style) check prepare&open before every VExpr execute

### PR DESCRIPTION
**Pull request by zclllyybb**
## Proposed changes

Issue Number: close #xxx

added `DCHECK` for call chain: `prepare->open->execute`.
fixed:
1. open for slot ref
2. open for point query
3. alloc_resource for set node

btw, fixed many code style.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

File: be/src/vec/exprs/vruntimefilter_wrapper.cpp [hunk 1]
```diff
@@ -49,14 +49,20 @@ VRuntimeFilterWrapper::VRuntimeFilterWrapper(const TExprNode& node, const VExprS
 
 Status VRuntimeFilterWrapper::prepare(RuntimeState* state, const RowDescriptor& desc,
                                       VExprContext* context) {
-    RETURN_IF_ERROR_OR_PREPARED(_impl->prepare(state, desc, context));
+    RETURN_IF_ERROR_OR_PREPARED(VExpr::prepare(state, desc, context));
```
[the remark this answers is not in the document]
Comment (zclllyybb) [reply be/src/vec/exprs/vruntimefilter_wrapper.cpp]: no need to prepare itself

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Review by BiteTheDDDDt — DISMISSED** [review]

**Comment by zclllyybb** [comment]
run buildall

**Review by BiteTheDDDDt — APPROVED** [review]

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**